### DISCUSSION TITLE
How Do You know What Builds Have Library's

### DISCUSSION THREAD

**Discussion by dennisdballen** [community/community#113583] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Issues

### Body

To Whom It May Concern, 
Why Does It Say That The 
(Xerox Plus Build) Has A Library With The Build On It 
When It Doesn't  ?
And What Kodi Builds Have Library's To Save Content  !
Please Send Me Some Of The Best Builds That Work On Kodi 20.5 Version  !
Thank You Very Much For All Your Help And Time And Support  !
      Sincerely, 
Dennis DB Allen 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @dennisdballen ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on those builds. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
